### PR TITLE
UX: Multiple codeblocks fixes

### PR DESCRIPTION
**Pull request by jordanvidrine**
This PR addresses a couple issues:

- lines up inline codeblocks with web standard styling
- adds a darker bg for inline codeblock for better readability

### After
image

### Before
image

### CODE REVIEW

File: app/assets/stylesheets/common/base/code_highlighting.scss [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-    Menlo, Monaco, Consolas, monospace;
+  padding: 2px 4px;
+  background: var(--inline-code-bg);
+  white-space: pre-wrap;
+  color: var(--primary);
+}
+
+code {
   color: var(--primary-very-high);
   background: var(--hljs-bg);
```
Comment (pmusaraj) [inline app/assets/stylesheets/common/base/code_highlighting.scss:13]: So the `--hljs-bg` color applies now to all code blocks except for those right below `p`, `em`, `li`... it's a little confusing.

File: app/assets/stylesheets/common/base/code_highlighting.scss [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
   border-radius: var(--d-button-border-radius);
   font-size: 14px;
   line-height: calc((13 + 4) / 13);
 }
 
-.hljs {
+pre > code {
```
Comment (pmusaraj) [inline app/assets/stylesheets/common/base/code_highlighting.scss:19]: If I read this correctly, `pre > code` now has different padding to `code` under a non-pre parent (2px 4px padding above, 12px here for `pre > code`.

**Review by pmusaraj — COMMENTED** [review]
Maybe there should be before/after screenshots here? I am finding it difficult to follow all the repercussions of the changes.

File: app/assets/stylesheets/common/base/code_highlighting.scss (hunk 1 above)
Comment (jordanvidrine) [reply app/assets/stylesheets/common/base/code_highlighting.scss:13]: I've added a screenshot. The large codeblocks will have their own background, and any inline codeblocks will have a *slightly* darker background. This is how stack overflow also renders their codeblocks. The bg color is different in linline versus a whole block.

File: app/assets/stylesheets/common/base/code_highlighting.scss (hunk 2 above)
Comment (jordanvidrine) [reply app/assets/stylesheets/common/base/code_highlighting.scss:19]: I have specified the padding to be different for inline versus "block" code.

**Review by pmusaraj — APPROVED** [review]

**Pull request merged**